diff --git a/modules/order/src/Plugin/views/area/OrderTotal.php b/modules/order/src/Plugin/views/area/OrderTotal.php
index bfebab86..892fbad3 100644
--- a/modules/order/src/Plugin/views/area/OrderTotal.php
+++ b/modules/order/src/Plugin/views/area/OrderTotal.php
@@ -80,6 +80,7 @@ class OrderTotal extends AreaPluginBase {
           'commerce_order.order_id',
           'commerce_order_item.order_id',
           'commerce_payment.order_id',
+          'commerce_order_item_registration.order_id',
         ])) {
           continue;
         }
