diff --git a/modules/payment/src/Plugin/Commerce/CheckoutPane/PaymentInformation.php b/modules/payment/src/Plugin/Commerce/CheckoutPane/PaymentInformation.php
index aeacbc82..cc3bbafe 100644
--- a/modules/payment/src/Plugin/Commerce/CheckoutPane/PaymentInformation.php
+++ b/modules/payment/src/Plugin/Commerce/CheckoutPane/PaymentInformation.php
@@ -160,6 +160,10 @@ class PaymentInformation extends PaymentCheckoutPaneBase {
    * {@inheritdoc}
    */
   public function buildPaneForm(array $pane_form, FormStateInterface $form_state, array &$complete_form) {
+    // Reload the order to ensure that price adjustments down to zero result in
+    // isZero() === TRUE.
+    $this->setOrder($this->entityTypeManager->getStorage('commerce_order')->load($this->order->id()));
+
     if ($this->collectBillingProfileOnly()) {
       // No payment is needed if we don't require payment method collection,
       // and the order balance is zero. In that case, collect just the billing
